src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { useState } from "react";
+
+import bem from "../../utils/bem";
+import Button from "../Button/Button";
+import { ButtonTypes } from "../Button/ButtonTypes";
+import Heading from "../Heading/Heading";
+import Icon from "../Icons/Icon";
+import { IconColors, IconNames } from "../Icons/IconTypes";
+import { NotificationTypes } from "./NotificationTypes";
+
+export interface NotificationProps {
+  /** BlockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
+  blockName?: string;
+  /** Optional prop to control horizontal alignment of the `Notification` content */
+  centered?: boolean;
+  /** Optional `className` that appears in addition to "notification" */
+  className?: string;
+  /** Optional prop to control whether a `Notification` can be dismissed (closed) by a user */
+  dismissible?: boolean;
+  /** Optional custom `Icon` that will override the default `Icon` */
+  icon?: React.ReactNode;
+  /** ID that other components can cross reference for accessibility purposes */
+  id?: string;
+  /** Optional prop to control the margin around the `Notification` component */
+  noMargin?: boolean;
+  /** Optional prop to control the coloring of notification text and the visibility of an applicable icon */
+  notificationType?: NotificationTypes;
+  /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
+  modifiers?: string[];
+}
+
+// NotificationHeading child-component
+export function NotificationHeading({ ...props }) {
+  const { children, className } = props;
+  return (
+    <Heading
+      level={4}
+      className={bem("notification-heading", [], "", [className])}
+    >
+      {children}
+    </Heading>
+  );
+}
+
+// NotificationHeading child-component
+export function NotificationContent({ ...props }) {
+  const { children, className } = props;
+  return (
+    <div className={bem("notification-content", [], "", [className])}>
+      {children}
+    </div>
+  );
+}
+
+export default function Notification(
+  props: React.PropsWithChildren<NotificationProps>
+) {
+  const {
+    blockName,
+    centered,
+    children,
+    className,
+    dismissible = false,
+    noMargin = false,
+    icon,
+    id,
+    modifiers = [],
+    notificationType = "standard",
+  } = props;
+
+  const [isOpen, setIsOpen] = useState(true);
+
+  if (!isOpen) {
+    return null;
+  }
+
+  const handleClose = () => setIsOpen(false);
+
+  const baseClass = "notification";
+
+  const notificationModifiers = modifiers ? modifiers : [];
+
+  notificationModifiers.push(notificationType);
+
+  if (centered) notificationModifiers.push("centered");
+
+  if (noMargin) notificationModifiers.push("no-margin");
+
+  const iconElement = () => {
+    if (icon) return icon;
```

Comment:
This is fine and I would not change this, but in general, I think using global scope props in functions can be buggy. Sometimes it's better to pass them all in as arguments. Don't update this though as it can be nitpicky.

Reply:
Got it.  Thanks.